### PR TITLE
enh(release): Prepare Centreon Gorgone 20.04.11 RN

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

Please include a short resume of the changes and what is the purpose of the PR.

Any relevant information should be added to help reviewers.

## Target serie

- [x] 20.04.x
- [ ] 20.10.x
- [ ] 21.04.x
- [ ] 21.10.x (master)

### CODE REVIEW

File: fr/releases/centreon-core.md [hunk 1]
```diff
@@ -1326,6 +1326,17 @@ This patch fixes that.
 
 ## Centreon Gorgone
 
+### 20.04.11
+
+#### Correctifs
+
+- [anomaly] Le host id était null dans les filtres de detection.
```
Comment (cg-tw) [inline fr/releases/centreon-core.md:1333]: ID, détection

File: en/releases/centreon-core.md [hunk 2]
```diff
@@ -1324,6 +1324,17 @@ This patch fixes that.
 
 ## Centreon Gorgone
 
+### 20.04.11
+
+#### Bugfixes
+
+- [anomaly] Host id was null in detection filters
```
Comment (cg-tw) [inline en/releases/centreon-core.md:1331]: ID

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: en/releases/centreon-core.md (hunk 2 above)
Comment (tuntoja) [inline en/releases/centreon-core.md:1331]:
```suggestion
- [anomaly] Host ID was null in detection filters
```

File: en/releases/centreon-core.md [hunk 3]
```diff
@@ -1324,6 +1324,17 @@ This patch fixes that.
...
+
+### Enhancements
```
Comment (tuntoja) [inline en/releases/centreon-core.md:1333]:
```suggestion
#### Enhancements
```

File: fr/releases/centreon-core.md [hunk 4]
```diff
@@ -1326,6 +1326,17 @@ This patch fixes that.
...
+
+### Améliorations
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:1335]:
```suggestion
#### Améliorations
```

File: fr/releases/centreon-core.md (hunk 1 above)
Comment (tuntoja) [inline fr/releases/centreon-core.md:1333]:
```suggestion
- [anomaly] Le host ID était null dans les filtres de détection.
```

File: en/releases/centreon-core.md [hunk 5]
```diff
@@ -1324,6 +1324,17 @@ This patch fixes that.
 
 ## Centreon Gorgone
 
+### 20.04.11
+
+#### Bugfixes
+
+- [anomaly] Host ID was null in detection filters
```
Comment (omercier) [inline en/releases/centreon-core.md:1331]:
```suggestion
- [Anomaly] Host ID was null in detection filters
```

File: en/releases/centreon-core.md [hunk 6]
```diff
@@ -1324,6 +1324,17 @@ This patch fixes that.
...
+
+#### Enhancements
+
+- add logging option to better handle logging that could cause SQLite database to grow too large
```
Comment (omercier) [inline en/releases/centreon-core.md:1335]:
```suggestion
- Add logging option to better handle logs that could cause SQLite database to grow too large
```

File: en/releases/centreon-core.md [hunk 7]
```diff
@@ -1324,6 +1324,17 @@ This patch fixes that.
...
+- added dependency for perl-Libssh-Session-0.8
```
Comment (omercier) [inline en/releases/centreon-core.md:1336]:
```suggestion
- Added dependency for perl-Libssh-Session-0.8
```

File: fr/releases/centreon-core.md [hunk 8]
```diff
@@ -1326,6 +1326,17 @@ This patch fixes that.
 
 ## Centreon Gorgone
 
+### 20.04.11
+
+#### Correctifs
+
+- [anomaly] Le host ID était null dans les filtres de détection.
```
Comment (omercier) [inline fr/releases/centreon-core.md:1333]:
```suggestion
- [Anomaly] Le host ID était null dans les filtres de détection.
```

File: fr/releases/centreon-core.md [hunk 9]
```diff
@@ -1326,6 +1326,17 @@ This patch fixes that.
...
+
+#### Améliorations
+
+- ajout d'une option de journalisation pour éviter une taille trop grande de la base SQLite
```
Comment (omercier) [inline fr/releases/centreon-core.md:1337]:
```suggestion
- Ajout d'une option de journalisation pour éviter une taille trop grande de la base SQLite
```

File: fr/releases/centreon-core.md [hunk 10]
```diff
@@ -1326,6 +1326,17 @@ This patch fixes that.
...
+- ajout de la dépendance pour perl-Libssh-Session-0.8
```
Comment (omercier) [inline fr/releases/centreon-core.md:1338]:
```suggestion
- Ajout de la dépendance pour perl-Libssh-Session-0.8
```

**Review by omercier — CHANGES_REQUESTED** [review]

**Review by mdebauchez — APPROVED** [review]

**Review by thiuyendang — APPROVED** [review]

File: en/releases/centreon-core.md [hunk 11]
```diff
@@ -1324,6 +1324,17 @@ This patch fixes that.
 
 ## Centreon Gorgone
 
+### 20.04.11
+
```
Comment (tuntoja) [inline en/releases/centreon-core.md:1328]:
```suggestion

`June 10, 2021`

```

File: fr/releases/centreon-core.md [hunk 12]
```diff
@@ -1326,6 +1326,17 @@ This patch fixes that.
 
 ## Centreon Gorgone
 
+### 20.04.11
+
```
Comment (tuntoja) [inline fr/releases/centreon-core.md:1330]:
```suggestion

`10 juin 2021`

```

**Pull request merged**